modules/dns/variables.tf

```diff
@@ -55,3 +55,9 @@ variable "manage_subdomain" {
   default     = true
   type        = bool
 }
+
+variable "force_destroy" {
```

Comment:
We should name it the same to avoid confusion.

Reply:
Fixed